recipes/regreg/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "regreg" %}
+{% set version = "0.1.5" %}
+{% set sha256 = "d9d7d8f258d29ea55d57e4e8599236660837b05e04de96de36a4287cbf59b932" %}
+
+package:
+  name: regreg
+  version: {{ version }}
+
+source:
+  url: https://github.com/regreg/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
```

Comment:
Since this is using Cython and compiling an extension it can't be noarch:
```suggestion
```

Reply:
Thanks. So, after reading https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors I guess this line should just be deleted. (Must admit that line is there from following an example meta.yaml)

Reply:
Yup, that's what I was thinking. If you do that and everything works this should be ready to go.